CHANGELOG.md

```diff
@@ -24,7 +24,15 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/report/compare/release-3.2.0...master
+[Unreleased]: https://github.com/atlassian/report/compare/release-3.2.1...master
+
+## [3.2.1] - 2019-05-10
```

Comment:
Speculative release marking is meh, but ok

Reply:
What does "Speculative release marking" mean?

Reply:
f173777 speculates that `report:3.2.1` will be released:
* successfully
* on 2019-05-10
* with the given scope
* with the given version number

Contrast this with https://github.com/atlassian/infrastructure/pull/21 which marked the release after the fact, so all of these aspects were history rather than speculation.

Reply:
Clear. Thanks, will keep in mind for the future